### PR TITLE
worked on overall design of main page (top slider, live action user p…

### PR DESCRIPTION
**Pull request by hekmatnajafi**
…rofiles, hot collection, and explore part). sill working.

### CODE REVIEW

File: Components/explore.js [hunk 1]
```diff
@@ -1,7 +1,27 @@
 import styles from '/styles/Products.module.css'
```
Comment (prince1456) [inline Components/explore.js:1]: Hi please remove the css modules and use styled component instead its not good idea to use both in same time

**Review by prince1456 — CHANGES_REQUESTED** [review]

File: Components/explore.js (hunk 1 above)
Comment (hekmatnajafi) [reply Components/explore.js]: done, removed the module.css on the parts i worked and will remove the remaining modules when i worked on that specific part.
also, moved the constant texts to their specific file

**Comment by prince1456** [comment]
hekmat Jan you have 3 conflict please solve them

**Comment by hekmatnajafi** [comment]
done, what happened? it had no conflicts just minutes before.
maybe after mohins's merge, my branch got conflicts

**Comment by hekmatnajafi** [comment]
done, what happened? it had no conflicts just minutes before.
maybe after mohins's merge, my branch got conflicts

**Pull request merged**